docs/tdvt.md

```diff
@@ -544,3 +544,41 @@ Check that any <span style="font-family: courier new">date-format</span> element
 
 __The agg.countd expression test and the join.null.int logical tests are failing__
 Check that your database is correctly returning column nullability information in the metadata. See [Design Considerations]({{ site.baseurl }}/docs/design) for more information.
+
+## Running TDVT test with tabquerytoolsrv.exe
+You can run a particular TDVT test with the help of a test setup file using tabquerytoolsrv.exe. `tabquerytoolsrv.exe` is located in `C:\Program Files\Tableau\Tableau Server\packages\bin.<version_code>\tabquerytoolsrv.exe` for Windows and `/opt/tableau/tableau_server/packages/bin.<version_code>/tabquerytoolsrv` for Linux. 
+ Before being able to run TDVT tests using tabquerytoolsrv make sure that: 
+   - Your instance of the tableau server is running 
+   - Your server license has been activated 
+Once the above steps are verified you should be able to run TDVT tests using tabquerytoolsrv by running the following command:
+
+__For Windows__
+```
+ tabquerytoolsrv.exe --query-file-list D:\test\tests.txt -d D:\path_to_tds\cast_calcs.db.tds --combined --password-file D:\path_to_tds\db.password --output-dir D:\test -DLogDir=D:\test -DOverride=ProtocolServerNewLog -DLogicalQueryRewriteDisable=Funcall:RewriteConstantFuncall -DInMemoryLogicalCacheDisable -DParamFile="C:\ProgramData\Tableau\Tableau Server\data\tabsvc\config\minerva_0.<version_code>\workgroup.yml,C:\ProgramData\Tableau\Tableau Server\data\tabsvc\config\minerva_0.<version_code>\minerva.yml"
+Attempting to run query...
+Run query successful! Check output file
+```
+
+__For Linux:__
+```
+./tabquerytoolsrv  --query-file-list ~/test/tests.txt -d  ~/path_to_tds/cast_calcs.db.tds --combined --password-file ~/path_to_tds/db.password --output-dir ~/test -DLogDir=~/test -DOverride=ProtocolServerNewLog -DLogicalQueryRewriteDisable=Funcall:RewriteConstantFuncall -DInMemoryLogicalCacheDisable -DParamFile="/var/opt/tableau/tableau_server/data/tabsvc/config/minerva_0.<version_code>/workgroup.yml,/var/opt/tableau/tableau_server/data/tabsvc/config/minerva_0.<version_code>/minerva.yml
+Attempting to run query...
+Run query successful! Check output file
```

Comment:
Does this run all of TDVT or just a single test query? The goal of this work is to run TDVT on Linux, having to run each test query xml one by one is not a sufficient solution since they'd need to check that the test passed themselves, and they need to pick the correct input file which is frustrating. Plus, this only works for logical tests, expression tests do not have the inputs in logical query xml.

I haven't looked to much at tabquerysrv, so I may be wrong, but my understanding was that the goal was to have the tdvt.ini file [here](https://github.com/tableau/connector-plugin-sdk/blob/master/tdvt/tdvt/config/tdvt/tdvt.ini) point to tabquerysrv, which would allow users to run TDVT and TDVT would use tabquerysrv under the hood, like it does with tabquerytool for Desktop installations.

Reply:
Right now adding `tabquerytoolsrv` to `tdvt_override.ini` is not working. Looking into the logs looks like we might need to do some work to add param files in the ini file. I am unaware if we have already done this.

Reply:
Adding the `-DParamFile` using `CommandLineOverride` worked.